packages/react-components/react-toolbar/e2e/Toolbar.e2e.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { mount as mountBase } from '@cypress/react';
+
+import { FluentProvider } from '@fluentui/react-provider';
+import { teamsLightTheme } from '@fluentui/react-theme';
+
+import { Toolbar, ToolbarButton, ToolbarDivider } from '@fluentui/react-toolbar';
+
+const mount = (element: JSX.Element) => {
+  mountBase(<FluentProvider theme={teamsLightTheme}>{element}</FluentProvider>);
+};
+
+describe('MenuTrigger', () => {
+  it('should focus first element', () => {
+    mount(
+      <Toolbar>
+        <ToolbarButton>Item 1</ToolbarButton>
+        <ToolbarButton>Item 2</ToolbarButton>
+        <ToolbarDivider />
+        <ToolbarButton>Item 3</ToolbarButton>
+      </Toolbar>,
+    );
+    cy.get('[role="toolbar"]').type('{rightarrow}').get('[type="button"]').eq(0).should('be.focused');
```

Comment:
I don't understand this test, you are focusing the entire toolbar container and then pressing right arrow key ? it doesn't make sense since you would never focus on the toolbar container in reality.

I'd rather do a `realPress` with tab key and then start using arrow keys

Reply:
Done, pressing tab from the `html` element, couldn't make the `realPress` work from beginning